OMPython/__init__.py

```diff
@@ -238,8 +239,11 @@ def sendExpression(self, command):
             self._omc = None
             return result
           else:
-            answer = OMTypedParser.parseString(result)
-            return answer
+            try:
+              answer = OMTypedParser.parseString(result)
+              return answer
+            except:
```

Comment:
well, this is then just confusing nomenclature. this code snippet tries to parse a string called `result`. Let's consider this an **input** of this snippet, or "the string you want to parse" by passing it to a function called `parseString`. What you want to return is an `answer`, let's consider this an **output** of this snippet. 

Now, if `parseString` throws an exception, instead of reporting or returning the exception message, you return `result`, i.e. the **input** of your snippet. You drop all notice of an error having occured, and return an unparsed input, instead of a parsed output.

Btw, I don't know how familiar you are with python, but throwing exception is typically considered "proper error reporting", so the exceptions should contain valuable error information.